### PR TITLE
Fix documentation of extend + spelling.

### PR DESCRIPTION
**Pull request by luca-heltai**
Followup to #7914. Fix @bangerth comments.

### CODE REVIEW

File: include/deal.II/base/bounding_box.h [hunk 1]
```diff
@@ -141,7 +141,8 @@ class BoundingBox
   point_inside(const Point<spacedim, Number> &p) const;
 
   /**
-   * Increase (or decrease) the size of the bounding box by the given amount.
+   * Increase (or decrease) the size of the bounding box by the given amount,
+   * in both directions in all coordinate directions.
```
Comment (jppelteret) [inline include/deal.II/base/bounding_box.h]: Did you accidentally leave in text for the 2d case before generalising it?

`in both directions in all coordinate directions.` -> `in all coordinate directions.`

I would also suggest removing the comma at the end of the preceding line.

Comment (luca-heltai) [reply include/deal.II/base/bounding_box.h]: that was the suggestion by Wolfgang... :D both directions = +/-, in all coordinate directions: on x, y, z

Comment (jppelteret) [reply include/deal.II/base/bounding_box.h]: The new wording is much clearer now. Thanks a lot!

**Pull request merged**